configure.ac

```diff
@@ -21,6 +21,10 @@ AC_CANONICAL_HOST
 AC_PROG_INSTALL
 AC_PROG_CXX()
 
+AC_ARG_ENABLE([samples],
+    AS_HELP_STRING([--disable-samples, skip building of sample programs]),[], [enable_samples=yes])
+AM_CONDITIONAL([BUILD_SAMPLES], [test "$enable_samples" = "yes"])
```

Comment:
This flag should be named `--disable-examples` because our folder is named `example` and not `sample`.